plugin/trino-singlestore/src/test/java/io/trino/plugin/singlestore/TestingSingleStoreServer.java

```diff
@@ -31,23 +31,29 @@
 public class TestingSingleStoreServer
         extends JdbcDatabaseContainer<TestingSingleStoreServer>
 {
-    public static final String DEFAULT_VERSION = "7.8";
-    public static final String LATEST_TESTED_VERSION = "8.9";
+    // https://docs.singlestore.com/db/v9.0/support/singlestore-software-end-of-life-eol-policy
+    public static final String DEFAULT_VERSION = "8.7";
```

Comment:
Please also add in the description of the PR the symptoms you are seeing on your local machine that motivated this change.